### ISSUE TITLE
Adjust district_heating_shares() for the intermediate years on scenario path

### ISSUE THREAD

**Issue by ulfmueller** [openego/powerd-data#307]
Nutzen der Status Quo Werte aus p-e (basierend auf hotmaps) zur Interpolation mit eGon100RE – Werten
Take the status quo data from pypsa-eur ([see here](https://github.com/PyPSA/PyPSA-Eur-Sec/pull/149)) interpolation towards our data of the eGon100RE scenario.

**Comment by ClaraBuettner** [comment]
In case we do not run the update_heat_demand_timeseries function, the district heating shares in Germany are not adjusted.
So if we keep it that way, we should also update the district heating shares for Germany in this function. This is not part of the input file for district heating shares yet.

**Comment by ulfmueller** [comment]
I checked our approach and compared it with the data/approach from p-e. p-e has country-specific potentials and then assumes an overall country-unspecific progress over the years coming from today's country-specific values from hotmaps. Since countries like DK have higher potentials and status quo shares than e.g. XK I think the direction is quite similar to the hotmaps one we have for the eGon100RE. I think the added-value to manipulate the data is pretty low. I would vote for skipping this task entirely for the entire scenario path. Do you agree?

**Comment by ulfmueller** [comment]
I think our custom-made approach originated before p-e added the country-specific data from hotmaps...

**Comment by ClaraBuettner** [comment]
Yes, I also remember that data was coming from hotmaps for the other countries. 
For Germany the district heating share in pypsa-eur is about 42% (198 TWh) for 2045 which is higher than in the NEP. 
For me, it would be fine to keep it like this and probably adjust sth. in case the results are far away from the NEP. 
I can implement it on the same branch where I will kick out the scaling of heat demand timeseries.

**Comment by ClaraBuettner** [comment]
We agreed to not adjust the district heating shares from pypsa-eur. 
This is implemented in https://github.com/openego/powerd-data/tree/features/%23246-myopic-pypsaeur
